test/integration/model/upsert.test.js

```diff
@@ -598,6 +598,168 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           });
         });
       }
+
+
+      if (current.dialect.supports.inserts.onConflictWhere) {
+        describe('conflictWhere', () => {
+          const Users = current.define(
+            'users',
+            {
+              name: DataTypes.STRING,
+              bio: DataTypes.STRING,
+              isUnique: DataTypes.BOOLEAN
+            },
+            {
+              indexes: [
+                {
+                  unique: true,
+                  fields: ['name'],
+                  where: { isUnique: true }
+                }
+              ]
+            }
+          );
+
+          beforeEach(async () => Users.sync({ force: true }));
```

Comment:
Just a small thing, but there's no need for the `async` keyword, if you're not awaiting the `Users.sync` call

Reply:
:man_facepalming: oops, forgot to remove - I originally did await it as I had originally defined the model in the beforeEach.